backend/dal/Migrations/v01.03.00/Down/PostDown/01-NotificationTemplates.sql

```diff
@@ -0,0 +1,10 @@
+PRINT 'Updating NotificationTemplates'
+
+UPDATE dbo.[NotificationTemplates]
+SET [Body] = '
+@using System.Linq
+@using Pims.Dal.Entities
+<html><head><title>@Model.Environment.Title</title></head>
+<body><p>Dear @Model.Project.Manager,</p><p>Your project @Model.Project.ProjectNumber has been updated with the following note;</p><p>@Model.Project.PublicNote</p></body></html>'
```

Comment:
Fixing template now that the note is part of a collection instead of on the project.